### PR TITLE
Penitence Route, Grand Corridor (Cathedral F3 and F4)

### PR DESCRIPTION
**Pull request by vtange**

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -2566,7 +2566,7 @@ ETC_20150317_002565	And about in the paint of the meat
 ETC_20150317_002566	{memo X}Need more meats
 ETC_20150317_002567	{memo X}Feel hunger and have a headache
 ETC_20150317_002568	{memo X}I'm full
-ETC_20150317_002569	Giving meat to pilgrims
+ETC_20150317_002569	Giving meat to pilgrim
```
Comment (Sunaries) [inline ETC.tsv]: the pilgrim?

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -4870,26 +4870,26 @@ QUEST_LV_0200_20150323_004868	It seems that you are now more closer to the true
...
-QUEST_LV_0200_20150323_004883	Fortunately, the divine symbol of the spiritual power seems to be okay. Talk to Bishop Aurelius again.
-QUEST_LV_0200_20150323_004884	Recharge the divine symbol of the spiritual power
-QUEST_LV_0200_20150323_004885	Recharge the magical power after defeating the monsters nearby using the divine symbol of the spiritual power. If you have a difficulty recharging the symbol, get help from Bishop Aurelius.
-QUEST_LV_0200_20150323_004886	Obtain the third key of Maven at Veinika altar
-QUEST_LV_0200_20150323_004887	Obtain the third key of Maven using the divine symbol of the spiritual power at Veinika Altar.
-QUEST_LV_0200_20150323_004888	Talk to the disciple, Ruodell
-QUEST_LV_0200_20150323_004889	The disciple, Ruodell needs your help. Talk to the disciple, Ruodell.
-QUEST_LV_0200_20150323_004890	Look for the documents that are related to Naktis
-QUEST_LV_0200_20150323_004891	You've found the documents that would help removing the curse of Naktis. Talk to the disciple, Ruodell.
+QUEST_LV_0200_20150323_004872	$You've obtained Maven's First Key. Talk to Aurelius who is waiting at the Grand Corridor.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:4873]: Bishop Aurelius

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**